<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_home">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>456</width>
        <height>177</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_camera">
     <widget class="QLabel" name="label_img">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>601</width>
        <height>371</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>171</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Camera Set</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_opencam">
         <property name="text">
          <string>获取摄像头</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btn_closecam">
         <property name="text">
          <string>关闭摄像头</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_startcap">
         <property name="text">
          <string>开始采集</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="cbox_optcam"/>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="cbox_optfmt"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_network">
     <widget class="QGroupBox" name="groupBoxTCP">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>160</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>TCP Server</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_tcp">
         <property name="text">
          <string>TCPPort:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_tcpport">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>8888</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_disconn">
         <property name="text">
          <string>断开连接</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_connarm">
         <property name="text">
          <string>开启监听</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="textEdit_recv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>461</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_send">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>461</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_recv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Recv Display:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_send">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Display:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_senddata">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cleardata">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxUDP">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>UDP Server</string>
      </property>
      <widget class="QLabel" name="label_udp">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>55</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>UDPort:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_udpport">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>66</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>8090</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_home">
       <property name="text">
        <string>主  界  面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_camset">
       <property name="text">
        <string>相 机 设 置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_netset">
       <property name="text">
        <string>网 络 设 置</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
